docker-compose.yml

```diff
@@ -33,6 +33,10 @@ services:
       - snjs
 
   syncing-server-proxy:
+    depends_on:
+      - syncing-server-ruby
+      - syncing-server-js
+      - api-gateway
```

Comment:
not sure if I got all the dependencies right, but with many tries I was unable to crash the setup on bootup -- whereas previously it did sometimes, if the dependencies booted in the wrong order

Reply:
`depends_on` does not guarantee that the process in the container is started - it only secures the boot sequence of the containers.

Please use `wait-for.sh` for this purpose

Reply:
Addressed -- to make this work with nginx I had to create a Dockerfile for it.
Had trouble running it -- turns out there was a subtle bug in `wait-for.sh`, a consequence of the fact that in bash `exec $cmd` is not the same thing as `exec "$@"`, even if `cmd="$@"`.

Reply:
hmm this change is not necessary I think ... as nginx will be out of the picture.

These 2 PRs are very tighly coupled as it turns out: https://github.com/standardnotes/snjs/pull/261

Not sure yet which one should go first